### ISSUE TITLE
Open Contracting Data Standard

### ISSUE THREAD

**Issue by robscottd** [canada-ca/Open_First_Whitepaper#14]
The Public Works and Government Services Canada (PWGSC) Open Contracting Data Standard (OCDS) Pilot has been around for several years yet has not been advanced or roll out in a meaningful way.  The U.K. is embracing it and it can go along way to helping transparency and e-procurement initiatives.

Recent changes in the OCDS 1.1 release to better handle org structures and amendments make it even more applicable to Federal e-procurement and open standard initiatives.

How can we elevate OCDS to be the showcase Open Standard initiative (like it is in the U.K.)?

**Comment by smellems** [comment]
This is a great example!  Thanks you!
https://buyandsell.gc.ca/procurement-data/open-contracting-data-standard-pilot

**Comment by kenmcmillan** [comment]
Entirely agree. The culture at PSP does not appear to have absorbed what it got with OCDS. I would love to see a re-invigoration of this work. (But I'm a teensy bit biased since I was a contributor between 2010 and 2016.)

**Comment by Acasovan** [comment]
Great reminder @robscottd for us to add the OCDS as an example into this paper. When we were developing the data standard for common proactive disclosure reporting on [Open.Canada.ca](http://open.canada.ca/en/search/contracts) we tried to follow the standard as much as possible, additionally, we did a mapping to understand the gaps and the reasons for not being able to fully adopt it and have been working with OCDS to add additional items for more complete reporting. Canada has been doing some excellent work! We do need to do a better job of talking about it. Hopefully this paper will be one of those opportunities. Will leave this thread open until I update that section. Thanks!!!
